components/IPAScheduler.tsx

```diff
@@ -0,0 +1,97 @@
+"use client"
+
+import React from "react"
+import { Card, CardContent, CardTitle } from "@/components/ui/card"
+import { SectionHeader } from "@/components/ui/section-header"
+import { GiDna2 } from "react-icons/gi"
+import CCVBars from "@/components/assets/CCVBars"
+import { Button } from "@/components/ui/button"
+
+const request_access_url =
+  "https://docs.google.com/forms/d/e/1FAIpQLSdJwbhtMznPjR6QrobBG1u-n3AMLnWdigN-NqFepsC7gbb2tw/viewform"
+
+export const IPAScheduler = () => {
+  return (
+    <section className="mt-24 mb-24">
+      {/* <SectionHeader title="Ingenuity Pathway Analysis (IPA)" align="center" /> */}
+      <div className="flex justify-center items-center h-full w-full mx-2 px-4 sm:px-8 lg:px-24">
+        <div className="flex flex-col lg:flex-row items-center justify-center w-full max-w-[2880px] gap-8 sm:gap-12 lg:gap-20">
```

Comment:
Try this out and see what you like more? Basically this would make the card start at the top rather than centered vertically

```suggestion
        <div className="flex flex-col lg:flex-row items-start justify-center w-full max-w-[2880px] gap-8 sm:gap-12 lg:gap-20">
```

Reply:
Right now if the text changes, then the negative margin would also need to be updated because the card is centered vertically right now. If we do `items-start` and do a positive margin, it wouldn't matter if the text changes. I'd recommend positive margins if possible, negative margins are okay they can be tricky sometimes